.github/workflows/php.yaml

```diff
@@ -36,3 +36,57 @@ jobs:
       - uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           use-verbose-mode: 'yes'
+
+  unit-tests:
+    name: "Unit tests"
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: ['5.6', '7.1', '7.2', '7.3', '7.4']
+        dependencies: ['']
+        include:
+          - { php-version: '5.6', dependencies: '--prefer-lowest' }
+          - { php-version: '8.0', dependencies: '--ignore-platform-req=php' }
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-version }}
+          extensions: mbstring, intl, zip
+          coverage: xdebug
+          ini-values: ${{ matrix.xdebug-ini-values }}
+          tools: composer:v2
+
+      - name: Install dependencies
+        run: composer update --no-interaction ${{ matrix.dependencies }}
+
+      - name: Apply PHPUnit pathes
+        if: ${{ matrix.php-version < 7.2 }}
+        run: scripts/apply-phpunit-patches.sh
+
+      - name: Run tests
+        run: vendor/bin/phpunit --testsuite unit --colors=always --coverage-clover ./logs/clover.xml
```

Comment:
```suggestion
        run: composer run test -- --testsuite unit --coverage-clover ./logs/clover.xml
```

Reply:
Is there some benefit in running it like this, via composer?

Reply:
I would say to be consistent with already defined commands
If you use the command on your dev setup then also use it on the CI to not have different results one day :p

Reply:
I see, but I look on the composer commands as "shortcuts" on syntax sugar for developers. Here we use specific settings for CI, so we want kind of "different" result (meaning different output for example).

Also as you mention below, it won't be good for long running tests, so maybe using directly the phpunit command everywhere on CI will be more consistent, in the end :)

Reply:
I agree with the consistency point ^^

Maybe I said this also because my way of learning is by reading code and not manuals, and having a CI check and use dev commands seemed better :)